tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -597,6 +617,8 @@ static void *spi_loopback_setup(void)
 	memcpy(buffer_tx, tx_data, sizeof(tx_data));
 	memset(buffer2_tx, 0, sizeof(buffer2_tx));
 	memcpy(buffer2_tx, tx2_data, sizeof(tx2_data));
+	memset(buffer3_tx, 0, sizeof(tx3_data));
+	memcpy(buffer3_tx, tx3_data, sizeof(tx3_data));
```

Comment:
```suggestion
	memset(buffer3_tx, 0, sizeof(buffer3_tx));
	memcpy(buffer3_tx, tx3_data, sizeof(tx3_data));
```
Otherwise, the `memset` does not make much sense.